CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.0.0] - 2023-05-22
 ### Changed
 
 - Don't export self replication status.
```

Comment:
Is there a cartridge-metrics-role release? In SDK? I think we should prepare this at first.

For now, its CI is red (the error is relatively easy to fix) https://github.com/tarantool/cartridge-metrics-role